### PR TITLE
Add colorized output. Fix missing configuration warning

### PR DESCRIPTION
**Pull request by dkkoval**

### CODE REVIEW

File: pkg/config/core.go [hunk 1]
```diff
@@ -24,10 +23,7 @@ func parseConfig() {
 	viper.AddConfigPath("/etc/kaigara")
 	viper.AddConfigPath("$HOME/.kaigara")
 
-	err := viper.ReadInConfig()
```
Comment (dkkoval) [inline pkg/config/core.go]: Here I deleted this warning, cause anyway if we want to use config, we will check for it.
When you run `kaigara --help` the warning about some missing configuration looks strange, I think

File: cmd/root.go [hunk 2]
```diff
@@ -9,14 +9,14 @@
...
-	"os"
-
+	"github.com/fatih/color"
+	"github.com/mod/kaigara/pkg/term"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
 var cfgFile string
+var noColor bool
```
Comment (mod) [inline cmd/root.go]: You should use pkg/config to store all options

File: cmd/root.go [hunk 3]
```diff
@@ -33,20 +33,20 @@ var RootCmd = &cobra.Command{
...
-		fmt.Println(err)
-		os.Exit(-1)
-	}
+	RootCmd.Execute()
 }
 
 func init() {
 	cobra.OnInitialize(initConfig)
 	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.kairc)")
+	RootCmd.PersistentFlags().BoolVar(&noColor, "no-color", false, "disable colorized output")
```
Comment (mod) [inline cmd/root.go]: I would like to disable it by default
you can enable it using --color and kairc file

File: pkg/term/output_test.go [hunk 4]
```diff
@@ -1,6 +0,0 @@
-package term
```
Comment (mod) [inline pkg/term/output_test.go]: Let's add test and default it to disabled
I would like to make it easy for people to have a .kairc file
we can try
kaigara config core.color true

Comment (mod) [inline pkg/term/output_test.go]: We need test here

**Pull request closed without merging**